homeassistant/const.py

```diff
@@ -51,6 +51,7 @@
 CONF_CUSTOMIZE = 'customize'
 CONF_CUSTOMIZE_DOMAIN = 'customize_domain'
 CONF_CUSTOMIZE_GLOB = 'customize_glob'
+CONF_DAYS = 'days'
```

Comment:
Do that only if you can replace it and use it on other components/platform

Reply:
What should I remove or how should I change it to be correct?

Reply:
I think @pvizeli is saying that `const.py` is for constants that are broadly applicable; as far as we know, `CONF_DAYS` is only useful to your platform at the moment. So, until it's determined otherwise, it's best placed within `spotcrime.py`.

Reply:
Thank you @bachya, I see now. I did not understand well enough how to bring in the configuration variables. Some of the information at this link gave me a better idea: https://community.home-assistant.io/t/help-using-config-get-for-custom-platform-switch/4092/2. I am working to do it the correct way now.